recipes/libattr/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+    homepage = "https://savannah.nongnu.org/projects/attr/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _pkg_etc(self):
+        return os.path.join(
+            self.package_folder,
+            "etc"
+        )
+
+    @property
+    def _pkg_lib(self):
+        return os.path.join(
+            self.package_folder,
+            "lib"
+        )
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+    
+    @property
+    def _inc_attr_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "include",
+            "attr"
+        )
+
+    @property
+    def _libs_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            ".libs"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(
+                "libattr is just supported for Linux")
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True,
+                  destination=self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(
+            self, win_bash=tools.os_info.is_windows)
+        conf_args = [
+            "--prefix={}".format(tools.unix_path(self.package_folder)),
+            "--bindir={}".format(tools.unix_path(
+                os.path.join(self.package_folder, "bin"))),
+            "--libdir={}".format(tools.unix_path(
+                os.path.join(self.package_folder, "lib")))
+        ]
+        if self.options.shared:
+            conf_args.extend(["--enable-shared", "--disable-static"])
+        else:
+            conf_args.extend(["--disable-shared", "--enable-static"])
+        self._autotools.configure(args=conf_args)
+        return self._autotools
+
+    def build(self):
+        with tools.chdir(self._source_subfolder):
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        with tools.chdir(self._source_subfolder):
+            autotools = self._configure_autotools()
+            autotools.install()
+        tools.rename(
+            os.path.join(self._pkg_etc, "xattr.conf"),
+            os.path.join(self._pkg_lib, "xattr.conf")
```

Comment:
```suggestion
            os.path.join(os.path.join(self.package_folder, "res", "xattr.conf"))
```

Use `res` folder instead for configuration files, it's mapped and can be easily handled by Conan.

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info
https://docs.conan.io/en/latest/reference/generators/text.html

Reply:
once again thanks for enlightment. i will keep in mind. i changed according to ur suggest.